omnigibson/termination_conditions/predicate_goal.py

```diff
@@ -44,3 +44,12 @@ def goal_status(self):
                 of the predicates matching either of those conditions
         """
         return self._goal_status
+
+    @property
+    def partial_success(self):
+        """
+        Returns:
+            bool: returns true if partial success is supported, false otherwise
+        """
+        assert self._done is not None, "At least one step() must occur before partial_success can be calculated!"
```

Comment:
You are not computing partial success here though, only whether this termination condition supports partial success or not. Do you still need to require `done is not None`?